app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -240,6 +242,17 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
         when (val executionInfo = txDetails.detailedExecutionInfo) {
             is DetailedExecutionInfo.MultisigExecutionDetails -> {
                 binding.txConfirmations.visible(true)
+                binding.txConfirmButton.visible(true)
```

Comment:
why is the confirm button always visible?

Reply:
How do I know which owner address is stored in the device?

Reply:
see `OwnerCredentialsRepository`. If hasCredentials() returns true we can retrieve key and address with retrieveCredentials. For example check AppSettingsFragment and its view model.

Reply:
Thanks!